runtime/onert/core/include/ir/DataType.h

```diff
@@ -39,6 +39,8 @@ enum class DataType
   QUANT_INT16_ASYMM = 10,
   QUANT_INT8_SYMM_PER_CHANNEL = 11,
   QUANT_INT16_SYMM = 12,
+  QUANT_UINT4_SYMM_PER_BLOCK = 13, // Symmetric by substracting 8 always
+  QUANT_INT8_SYMM_PER_BLOCK = 14
```

Comment:
Please find my another comment in https://github.com/Samsung/ONE/pull/13693#discussion_r1732497335. I prefer the name like `QUANT_GGML_Q4_0` which specifies it is the same to `ggml_q4_0`. It may be helpful for someone who wants find the details of quantization type.

Reply:
I've renamed. PTAL